AuthenticationExample/AuthenticationExample/ProfileView.swift

```diff
@@ -28,15 +28,28 @@ struct ProfileView: View {
     /// The closure to call once the user has signed out.
     var signOutAction: () -> Void
     
+    /// Dismisses the current presentation.
+    @Environment(\.dismiss)
+    private var dismiss
+    
     var body: some View {
-        VStack {
-            if let user = portal.user {
-                UserView(user: user).padding()
+        NavigationStack {
+            VStack {
+                if let user = portal.user {
+                    UserView(user: user).padding()
+                }
+                Spacer()
+                signOutButton
+            }
+            .padding()
+            .toolbar {
+                ToolbarItem(placement: .topBarTrailing) {
+                    Button("Done") {
+                        dismiss()
+                    }
+                }
```

Comment:
All I did here was add a done button since on mac catalyst and visionOS there is no way to exit the profile view.